webapp/packages/core-sdk/src/queries/getAsyncTaskInfo.gql

```diff
@@ -13,6 +13,7 @@ mutation getAsyncTaskInfo(
         error {
             message
             errorCode
+            errorType
```

Comment:
please create `AsynkTaskInfoFragment` and use it in queries (like `exportDataFromContainer`)